client_test.go

```diff
@@ -69,15 +78,63 @@ func ExampleNewClient() {
 	}
 
 	// use a devnull connector for example purposes
-	conn := &devnull.Connector{}
+	conn, err := dosaRenamed.GetConnector("devnull", nil)
+	if err != nil {
+		panic(err)
```

Comment:
It's impossible for the devnull connector to return any errors, but I'll change it anyway.